dashboard/layout/pages/tmobile/project_view.py

```diff
@@ -1,25 +1,38 @@
 import dash_html_components as html
 
-from data.data import completed_status_counts
+import config
```

Comment:
Is de piechart ook veranderd in het KPN front end gedeelte? of komt dit nog in een ander ticket? (de use case van dit ticket doet vermoeden dat het om een generieke aanpassing gaat en niet T-Mobile specifiek)

Reply:
Dit word meegenomen in DAN-361